python/soma/spm/spm12/tools/shoot_tools/__init__.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import absolute_import
 from soma.spm.spm_main_module import SPM12MainModule
-from soma.spm.spm_batch_maker_utils import addBatchKeyWordInEachItem, moveSPMPath
-
+from soma.spm.spm_batch_maker_utils import moveSPMPath, convertlistToSPMString, convertNumpyArrayToSPMString, convertPathListToSPMBatchString
+from soma.spm.custom_decorator_pattern import checkIfArgumentTypeIsAllowed, checkIfArgumentTypeIsStrOrUnicode
+import numpy
+import numbers
```

Comment:
Imported but not used

Reply:
I remove numbers but the others are used, for some I am not sure what they do but I kept it like in other SPM brainvisa scripts.